Dockerfile

```diff
@@ -1,24 +1,19 @@
-# Use an official Node.js runtime as a parent image
 FROM node:16-alpine
 
-# Set the working directory
+# Instalar dependências de construção
+RUN apk add --no-cache python3 make g++
+
 WORKDIR /usr/src/app
 
-# Install app dependencies by copying
-# package.json and package-lock.json (or yarn.lock)
 COPY package*.json ./
 
-# Install dependencies
+# Instalar dependências, incluindo bcryptjs
```

Comment:
Code comments, variable names, etc should be always into the native languagem of the programming language. Most of the languages are written, in english, like javascript/typescript. 
Update your texts to english